orttraining/orttraining/test/python/orttraining_test_dort.py

```diff
@@ -216,7 +216,12 @@ def elementwise_model(tensor_x: torch.Tensor):
             tensor_q = tensor_p.relu()
             return tensor_q
 
-        local_backend = make_local_backend(dynamic=True, use_aot_autograd=False)
+        # TODO: Set use_aot_autograd=False. In order to decompose torch
+        # function calls to aten ops, we need to set
+        # user_aot_autograd=True because there is no decomposition in DORT
+        # anymore. A long-term fix will be brining # decomposition pass back
```

Comment:
```suggestion
        # anymore. A long-term fix will be brining decomposition pass back
```

Reply:
Why DORT no longer has decomposition pass? Have you tried https://github.com/pytorch/pytorch/blob/main/torch/onnx/_internal/fx/passes/functionalization.py#L19

```python
    module = passes.Functionalize(
        diagnostic_context,
        module,
        enable_dynamic_axes=options.dynamic_shapes,
        allow_fake_constant=options.fake_context is not None,
    ).run(*fx_module_args)
```

Reply:
Never try this. FYI, I see this pass uses `make_fx` which crashed inside DORT before and the solution was to use DecompositionInterpreter.